### PR TITLE
Issue #4316: move argLine to properties to let it be overriden from CLI for Travis localized builds

### PR DESCRIPTION
**Pull request by romani**
Issue #4316

### CODE REVIEW

**Comment by rnveach** [comment]
@romani The fix also has to be given to all maven commands in the shell script `./.ci/travis/travis.sh`.

File: .travis.yml [hunk 1]
```diff
@@ -84,42 +84,42 @@ matrix:
...
       env:
         - DESC="tests pt"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=pt -Duser.country=PT'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=pt -Duser.country=PT -Xms1024m -Xmx2048m'"
     # unit tests in Turkish locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests tr"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=tr -Duser.country=TR'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=tr -Duser.country=TR -Xms1024m -Xmx2048m'"
```
Comment (rnveach) [inline .travis.yml]: Instead of recreating the whole argLine, can we have a new property like `argLineExtra` and in the POM, append it to `argLine`?

Comment (romani) [inline .travis.yml]: make sense. done.

**Comment by romani** [comment]
`-X` was removed , as it is not required now, as we know a reason of failure.
it even create too big log, so Travis have problems to show it sometimes.

**Comment by romani** [comment]
merged manually

**Pull request closed without merging**